liberapay/security/authentication.py

```diff
@@ -14,6 +17,9 @@
 from liberapay.models.participant import Participant
 from liberapay.utils import get_ip_net
 
+import passwordmeter
+from hashlib import sha1
+import requests
```

Comment:
Please don't break the import style like this. Imports are split into specific groups (based on [PEP8](https://www.python.org/dev/peps/pep-0008/#imports)), and ordered alphabetically within each group.